pom.xml

```diff
@@ -102,6 +102,13 @@
             <artifactId>mapstruct</artifactId>
             <version>${mapstruct.version}</version>
         </dependency>
+        <!--  force use of commons-lang3 version 3.18 to avoid vulnerability in 3.17 that is transitive
```

Comment:
Are you able to remove this forced dependency on commons-lang3 now that the other ch libraries have been upgraded?

Reply:
When I do `mvn dependency:tree` without this I still see `commons-lang3:jar:3.17.0` under `structured-logging`

The pom for `structured-logging` does actually state 3.18.0, so I'm a little puzzled.  I guess there's another dependency that has 3.17.0 and then it is used elsewhere.